### PR TITLE
rename jupyter notebook name

### PR DESCRIPTION
**Pull request by xunliu**
1. Rename `init/jupyter/gravitino-fileset-sample.ipynb` to `init/jupyter/gravitino-fileset-example.ipynb`
2. Improvement README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,12 +57,44 @@ cd gravitino-playground
...
+
 ```shell
 git clone git@github.com:datastrato/gravitino-playground.git
 cd gravitino-playground
 ./launch-playground.sh hive|gravitino|trino|postgresql|mysql|spark|jupyter
 ```
 
+### Experiencing Gravitino Fileset with Jupyter
+
+We provide a Fileset playground environment to help you quickly understand how to use Gravitino
```
Comment (qqqttt123) [inline README.md:69]: Could you avoid using `we`?

**Comment by qqqttt123** [comment]
Could you modify the document in our main repo?

**Comment by xunliu** [comment]
> Could you modify the document in our main repo?

Yes main repo already updated.

File: README.md (hunk 1 above)
Comment (xunliu) [reply README.md]: DONE

File: README.md [hunk 2]
```diff
@@ -54,12 +57,44 @@ cd gravitino-playground
...
 
 ### Launch special component or components of playground
+
 ```shell
 git clone git@github.com:datastrato/gravitino-playground.git
 cd gravitino-playground
 ./launch-playground.sh hive|gravitino|trino|postgresql|mysql|spark|jupyter
 ```
 
+### Experiencing Gravitino Fileset with Jupyter
```
Comment (qqqttt123) [inline README.md:67]: Could you provide a user case instead of an example?

**Pull request merged**